### ISSUE TITLE
VirtualTerminal crashes when using System LookAndFeel.

### ISSUE THREAD

**Issue by StochasticTinkr** [varabyte/kotter#117]
**Describe the bug**
When trying to use the VirtualTerminal, an error occurs if I've set the System LookAndFeel.

**To Reproduce**
Steps to reproduce the behavior.  Run this kotlin file.
```kotlin
import com.varabyte.kotter.terminal.virtual.VirtualTerminal
import java.awt.EventQueue.invokeLater
import javax.swing.UIManager

fun main() {
    invokeLater {
        UIManager.setLookAndFeel(UIManager.getSystemLookAndFeelClassName())
    }
    VirtualTerminal.create()
}
```
**Expected behavior**
A virtual terminal appears, decorated with the System look and feel.

**Screenshots**
```
java.lang.Error: no ComponentUI class for: com.varabyte.kotter.terminal.virtual.SwingTerminalPane[,0,0,0x0,invalid,alignmentX=0.0,alignmentY=0.0,border=,flags=0,maximumSize=,minimumSize=,preferredSize=,caretColor=,disabledTextColor=,editable=true,margin=,selectedTextColor=,selectionColor=,kit=,typeHandlers=]
...
Caused by: java.lang.NullPointerException: Cannot invoke "javax.swing.plaf.TextUI.getMinimumSize(javax.swing.JComponent)" because "ui" is null
```
and 
```
Exception in thread "AWT-EventQueue-0" java.lang.NullPointerException: Cannot invoke "javax.swing.plaf.TextUI.getMinimumSize(javax.swing.JComponent)" because "ui" is null
```

**Desktop:**
 - OS: MacOS 12.7.5
 - Kotter Version: 1.1.2
 - JDK 21.0.1-open
 - Kotlin 2.0.0

**Comment by bitspittle** [comment]
Hello, sorry to reply so late to your bug. (I think my email settings for Kotter filters must be messed up)

I just tried this:

```kotlin
fun main() {
    UIManager.setLookAndFeel(UIManager.getSystemLookAndFeelClassName())
    session(VirtualTerminal.create()) {
        section {
            textLine("HI")
        }.run()
    }
}
```
and also using `invokeLater`:
```kotlin
fun main() {
    invokeLater {
        UIManager.setLookAndFeel(UIManager.getSystemLookAndFeelClassName())
    }
    session(VirtualTerminal.create()) {
        section {
            textLine("HI")
        }.run()
    }
}
```
and I didn't see any exception in either case.

I am using a Mac at the moment, in case OS matters.

**Comment by bitspittle** [comment]
I created a new dummy project and was able to reproduce. Investigating.

**Comment by bitspittle** [comment]
OK, looks like I did something which resolved this issue in 1.2.0. (I bumped up Kotlin / JDK dependencies, so probably that)

**Comment by bitspittle** [comment]
1.2.0 is out. I'm going to close this issue but you can ping this bug if you're still using Kotter and still having the issue with 1.2.0...
